exporter/elasticsearchexporter/README.md

```diff
@@ -77,6 +77,6 @@ nodes will automatically be used for load balancing.
 ```yaml
 exporters:
   elasticsearch:
-    urls:
+    endpoints:
     - "https://localhost:9200"
```

Comment:
How does a deployment typically look like? Is it common to have TLS for one server, but not for another? Do they have different sets of certs, or would a client only need one CA that works for all clients? I'm asking because endpoints are typically only `host:port`, without the protocol part, using TLS by default.

Reply:
> Is it common to have TLS for one server, but not for another?

No, when using TLS one would want to have TLS for the complete cluster.

> Do they have different sets of certs, or would a client only need one CA that works for all clients?

Ideally one CA cert.

> I'm asking because endpoints are typically only host:port, without the protocol part, using TLS by default.

The exporter also accepts `localhost:9200`, but I'm not sure if TLS will be automatically enabled or disabled by default. I assume the client library will expand this to `http://localhost:9200` if tls is not explicitely configured.

I took [otlphttpexporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter) as example, which also uses a full URL in its example:

```
exporters:
  otlphttp:
    endpoint: https://example.com:55681/v1/traces
```

same for [zipkin](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/zipkinexporter#getting-started).

Reply:
@bogdandrutu ^

Interesting, the one from OTLP should have been named URL then, instead of Endpoint. Looks good to me then!